index.js

```diff
@@ -111,7 +111,12 @@ module.exports = opts => {
 	}
 
 	if (hasYarn(pkgCwd)) {
-		return execa('yarn', ['add', '--dev', 'xo'], {cwd: pkgCwd}).then(post);
+		return execa('yarn', ['add', '--dev', 'xo'], {cwd: pkgCwd})
+			.then(post)
+			.catch(() => {
+				const msg = "This project uses Yarn but you don't seem to have Yarn installed.\nRun 'npm install --global yarn' to install it."
```

Comment:
It should only show this message if the error is about missing Yarn, otherwise it should rethrow the error.

Reply:
Use single-quotes.

Reply:
Updated